documentation/asciidoc/topics/con_xsite_replication.adoc

```diff
@@ -0,0 +1,30 @@
+[id='global_clusters']
+= Cross-Site Replication with {ispn_operator}
+If you have {brandname} clusters running in separate locations, use
+{ispn_operator} to connect them so you can back up data across sites.
+
+For example, in the following illustration, {ispn_operator} manages a
+{brandname} cluster at a data center in New York City, **NYC**. At another data
+center in London, **LON**, {ispn_operator} also manages a {brandname} cluster.
+
+ifndef::productized[]
+image::xsite-ispn.svg[]
+endif::productized[]
+
+ifdef::productized[]
+image::xsite-rhdg.svg[]
+endif::productized[]
+
+{ispn_operator} uses a Kubernetes API to establish a secure connection between
+the {ocp} clusters in **NYC** and **LON**. {ispn_operator} then creates a cross-
+site replication service so {brandname} clusters can back up data across
+locations.
+
+{ispn_operator} selects one site master for each {brandname} cluster. The site
+master is the node that coordinates all backup requests through the cross-site
+replication service.
+
+If the current site master node becomes unavailable then {ispn_operator} finds
+a new coordinator node and makes it the site master. The cross-site replication
```

Comment:
I would remove the `coordinator` word and say the operator finds the newly elected SiteMaster.